### PR TITLE
[NCP-3449] some text added for showing history on detail page

### PR DESCRIPTION
**Pull request by jungho73**

### CODE REVIEW

**Review by burtonquantil — APPROVED** [review]
The English version looks OK

**Comment by Jianghai-Quantil** [comment]
@jungho73 Please do not haste to merge. The changes to texts in Chinese have not been reviewed yet. What @burtonquantil  approves is only the English version. Please invite Bin or me to review changes to Chinese texts and please always wait for our approval before merging changes. Thanks.

Some changes to the Chinese texts are required. Since this pull request is already merged, I'll use your new pull request 398 to request for changes.

**Pull request merged**
